<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Expressions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="ExpressionListView">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Variables</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="VariableListView">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Curves</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="CurveListView">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Press F1 for help</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QwtPlot" name="qwtPlot"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="actionExpression"/>
    <addaction name="actionVariable"/>
    <addaction name="actionCurve_from_DataSet"/>
    <addaction name="actionResponse_Plot"/>
    <addaction name="actionSystem"/>
    <addaction name="actionDiscrete_System"/>
    <addaction name="actionDiscrete_Continous_System"/>
    <addaction name="actionOpen_Sampling_Window"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionAutoscale"/>
   </widget>
   <widget class="QMenu" name="menuDocumentation">
    <property name="title">
     <string>Documentation</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionView_Control_System_Doc"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAdd"/>
   <addaction name="menuView"/>
   <addaction name="menuDocumentation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExpression">
   <property name="text">
    <string>Expression</string>
   </property>
  </action>
  <action name="actionVariable">
   <property name="text">
    <string>Variable</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>load</string>
   </property>
  </action>
  <action name="actionCurve_from_DataSet">
   <property name="text">
    <string>Curve from DataSet</string>
   </property>
  </action>
  <action name="actionResponse_Plot">
   <property name="text">
    <string>Response Plot</string>
   </property>
  </action>
  <action name="actionAutoscale">
   <property name="text">
    <string>autoscale</string>
   </property>
  </action>
  <action name="actionSystem">
   <property name="text">
    <string>Continous System</string>
   </property>
  </action>
  <action name="actionClosedLoop_System">
   <property name="text">
    <string>ClosedLoop System</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionView_Control_System_Doc">
   <property name="text">
    <string>View Control-System Doc.</string>
   </property>
  </action>
  <action name="actionDiscrete_System">
   <property name="text">
    <string>Discrete System</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionDiscrete_Continous_System">
   <property name="text">
    <string>Discrete-Continous System</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_Sampling_Window">
   <property name="text">
    <string>Open Sampling Window</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
